packages/shared/src/components/post/PostActions.tsx

```diff
@@ -180,9 +179,10 @@ export function PostActions({
             onClick={() => onCopyLinkClick(post)}
             icon={<LinkIcon />}
             responsiveLabelClass={actionsClassName}
-            className={shareExperience && 'hidden tablet:flex'}
-            variant={ButtonVariant.Tertiary}
-            color={ButtonColor.Cabbage}
+            className={classNames(
+              'btn-tertiary-cabbage',
+              shareExperience && 'hidden tablet:flex',
```

Comment:
This was not here before?

Reply:
It had: `className={shareExperience && 'hidden tablet:flex'}`